source/extensions/transport_sockets/tls/context_config_impl.cc

```diff
@@ -237,6 +241,10 @@ ContextConfigImpl::ContextConfigImpl(
   }
   capabilities_ = handshaker_factory->capabilities();
   sslctx_cb_ = handshaker_factory->sslctxCb(handshaker_factory_context);
+  if (!config.tls_keylog().tls_keylog_path().empty() && tls_keylog_src_.getIpListSize() == 0 &&
```

Comment:
Use the proto config to validate against so that you don't need to add `getIpListSize()`

Reply:
fixed